src/workloads/sharding/multi_updates/MultiUpdates.yml

```diff
@@ -1,5 +1,7 @@
 SchemaVersion: 2018-07-01
-Owner: "@mongodb/sharding"
+Owner: Cluster Scalability
+SlackSupportChannelName: "#server-sharding"
+SlackSupportChannelId: C8PK5KZ5H
```

Comment:
What's the reasoning for making this a raw ID? Having this as a URL seems more usable to me (e.g., https://mongodb.enterprise.slack.com/archives/C8PK5KZ5H) since I'm not aware in slack of any way of turning an ID into a URL without manually doing it.